test/inductor/test_torchinductor_opinfo.py

```diff
@@ -366,13 +366,13 @@ def wrapper_noop_set_seed(op, *args, **kwargs):
     # Following tests are failing with strict comparision but atol=1 is acceptable due roundings errors
     ("nn.functional.interpolate.bilinear", "cpu", u8): {"atol": 1, "rtol": 0},
     ("nn.functional.upsample_bilinear", "cpu", u8): {"atol": 1, "rtol": 0},
+    # atol ~10 is due to aten separable implementation using uint8 intermediate buffer and fixed point multiplication
+    ("nn.functional.interpolate.bicubic", "cpu", u8): {"atol": 10, "rtol": 0},
```

Comment:
This sounds very bad. Can't we fix this?

Reply:
This is an old threshold, now it can `atol=1` which is ok modulo rounding errors